Mathlib/Analysis/RCLike/Basic.lean

```diff
@@ -605,13 +610,23 @@ variable (K) in
 lemma nnnorm_nsmul [NormedAddCommGroup E] [NormedSpace K E] (n : ℕ) (x : E) :
     ‖n • x‖₊ = n • ‖x‖₊ := by simpa [Nat.cast_smul_eq_nsmul] using nnnorm_smul (n : K) x
 
+section NormedField
+variable [NormedField E] [CharZero E] [NormedSpace K E]
```

Comment:
do you want q and x here too?

Reply:
I don't like putting explicit variables in `variable`. That always catches me off guard, except when it's clearly used throughout the section, which would not be the case here

Reply:
Interesting personal preference, but okay